Cannot find module 'browserslist-config-single-spa'

This error comes up when building webpack apps that include single-spa-react. It's thrown from babel-loader. It was introduced in 4.5.0 - 4.4.0 works fine. I think that to fix it browserslist-config-single-spa should be moved from devDependencies to dependencies.

```
ERROR in ./node_modules/single-spa-react/lib/esm/single-spa-react.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Cannot find module 'browserslist-config-single-spa'
```

--- a/package.json
+++ b/package.json
@@ -60,6 +60,9 @@
     "url": "https://github.com/joeldenning/single-spa-react/issues"
   },
   "homepage": "https://github.com/joeldenning/single-spa-react#readme",
+  "dependencies": {
+    "browserslist-config-single-spa": "^1.0.1"
+  },
   "devDependencies": {
     "@babel/core": "^7.15.5",
     "@babel/eslint-parser": "^7.15.7",
@@ -75,7 +78,6 @@
     "@testing-library/react": "^13.0.0-alpha.1",
     "@types/react": "^17.0.24",
     "@types/react-dom": "^17.0.9",
-    "browserslist-config-single-spa": "^1.0.1",
     "concurrently": "^6.2.1",
     "copyfiles": "^2.4.1",
     "cross-env": "^7.0.3",

